src/lib/mongoBackend/mongoUpdateSubscription.cpp

```diff
@@ -488,20 +488,28 @@ static long long setLastFailure(const BSONObj& subOrig, CachedSubscription* subC
 
 /* ****************************************************************************
 *
-* setTimesFailed - 
+* setLastSuccess -
 */
-static void setTimesFailed(const BSONObj& subOrig, CachedSubscription* subCacheP, BSONObjBuilder* b)
+static long long setLastSuccess(const BSONObj& subOrig, CachedSubscription* subCacheP, BSONObjBuilder* b)
```

Comment:
Maybe a comment to setLastNotification() preamble should be added, e.g.:

"Note that this function returns void, different from setLastFailure() and setLastSucces() functions, which use long long. This is due to <a fragment based in your excelent explanation above>"